docs/device-management/device-management.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Device Management
+sidebar_position: 0
+---
+
+Device Management encompasses all of the services that allow you to control your
```

Comment:
Should add something about provisioning devices? Dylan has been calling out "Device Registry" in pricing documents as a separate service.

Reply:
Sure. But there's currently no documentation in this section about that so I think it should come in a future PR and not on this one.